#! /bin/sh /usr/share/dpatch/dpatch-run
## odbc.dpatch by Francesco Paolo Lovergine <frankie@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add library declaration to avoid tricky LIBS use at building time.
## DP: A better option would be fixing the configure.in to allow
## DP: user to specify ODBC dependencies.

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' proftpd-dfsg~/contrib/mod_sql_odbc.c proftpd-dfsg/contrib/mod_sql_odbc.c
--- proftpd-dfsg~/contrib/mod_sql_odbc.c	2010-04-19 11:46:04.000000000 +0200
+++ proftpd-dfsg/contrib/mod_sql_odbc.c	2010-04-19 12:13:13.000000000 +0200
@@ -22,6 +22,7 @@
  * the source code for OpenSSL in the source distribution.
  *
  * $Id: mod_sql_odbc.c,v 1.11 2009/10/02 21:22:56 castaglia Exp $
+ * $Libraries: -lodbc $
  */
 
 #include "conf.h"
